index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kalolo's Portfolio</title>
+    
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+    <nav class="nav-bar">
+
+    <div class="logo">
+        <a href="#">My logo</a>
+    </div>
+        <div class="menu-icon"> <img src="images/menu.png" alt="menu navbar"></div>
+    </nav>
+
+    <section class="first-section" id="section-one">
+        <div class="text">
+            <h1>
+                Hi I'm Kalolo,<br> I'm glad to see you!
+            </h1>
```

Comment:
- [ ] Let us make some modification here to arrange our heading, kindly, change the font-size or don't use `br` for the text (`I'm glad to see you!`).